styles/components/WorkSectionAsGallery.scss

```diff
@@ -0,0 +1,61 @@
+.WorkSectionAsGallery{
+    margin-top: 2rem; 
+
+    &__work-gallery-text {
+        padding: 0rem 1.25rem 0rem 1.25rem;
```

Comment:
why not use basement? `py0 px1_25 md:px5` etc.

fyi `padding: 0rem 1.25rem` is the same as `padding: 0rem 1.25rem 0rem 1.25rem`

Reply:
ahh so I didn't use basement because I figure that we were already using a stylesheet so might as well put it all in one place, to make things easier for the next developer to read :) how do you suggest approaching this in the future, when there's some styles that we can use basement with and some styles that we can't? should we just try and default to using basement / in-line classes whenever possible, and only put the styles that we can't use basement with in the stylesheet? I understand why this would make sense, but hopefully you can see the dilemma that I'm talking about, where if there's some styles that we can't use basement with, having styles in multiple places (i.e. as an in-line class and in the stylesheet) can get confusing. wdyt tho?

Reply:
@bbqxaxiu yeah we typically try and default to using basement / inline classes whenever possible, and only put the styles that we can't use basement with in the stylesheet